docs/benchmarks.md

```
@@ -199,24 +200,24 @@ before making a performance claim.
-| Interpreter | minivm `threaded` | 160.6 µs | 768 | 8 |
+| Interpreter | minivm `threaded` | 137.5 µs | 768 | 8 |
 |  | Tengo | 280.54 µs | 458,379 | 5,114 |
 |  | GopherLua | 545.66 µs | 818,512 | 11,253 |
 |  | Goja | 448.41 µs | 558,961 | 6,149 |
 |  | gpython | 1.26 ms | 2,570,669 | 34,797 |
 |  | Yaegi | 846.99 µs | 1,422,624 | 35,306 |
-| Native | minivm `default` | 169.1 µs | 768 | 8 |
-|  | minivm `jit` | 168.3 µs | 768 | 8 |
+| Native | minivm `default` | 133.5 µs | 768 | 8 |
+|  | minivm `jit` | 135.6 µs | 768 | 8 |
 | Reference | Native Go | 12.97 µs | 16,368 | 1,023 |
```

markdownlint-cli2 (0.23.2)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)